### ISSUE TITLE
Test requirement to start Android Studio after setup

### ISSUE THREAD

**Issue by LarkAscending** [flutter/flutter#9010]
## Steps to Reproduce

New user feedback on Android Setup was that it's necessary to start Android Studio after installing it, before running `flutter doctor`. 

Instructions here:
https://flutter.io/setup-windows/#install-android-studio

Test (on macOS, Windows, Linux):
1. On a clean machine, clone flutter repo and update path to include flutter/bin.
1. Install Android Studio (but don't start it).
1. Attach Android device.
1. `flutter doctor`

**Comment by eseidelGoogle** [comment]
@mit-mit can you answer this?

**Comment by LarkAscending** [comment]
A bit of an IT marathon was required, but here we go... tested on a Windows laptop w/connected Android device.

tl;dr Starting Android Studio after install is 100% required. Otherwise `flutter run` yields this:
`Unable to locate gradle. Please install Android Studio.`

`flutter doctor` runs fine but shows that 
* there's no JDK found (under Android toolchain) 
* Android Studio (not installed) - though it is.

Assuming this is also true on Mac and Linux, to the [setup instructions](https://flutter.io/setup-windows/) are correct. Starting Android Studio once after install is required for `flutter run`. 

Closing this issue.

**Comment by mit-mit** [comment]
@LarkAscending did you simply start Android Studio, or did you have to create an Android project, run it, or anything like that?

**Comment by LarkAscending** [comment]
Just started it - new installation, no project.
First time it starts it prompts you to upgrade. I said no thanks, then Next.
At which point it downloaded all kinds of stuff - took a couple of minutes. Unprompted...

Then `flutter run` after that was successful.

**Comment by mit-mit** [comment]
Got it, thanks!
